bin/tessel-install-drivers.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+var logs = require('../lib/logs');
+var child_process = require('child_process');
+
```

Comment:
@rwaldron isn't that what I have on [line 8](https://github.com/tessel/t2-cli/pull/416/files#diff-f195a6f2706c44203b9d76c77ba72d35R8)?